### PR TITLE
Fix surveys

### PR DESCRIPTION
**Pull request by DavidKutu**
remove experiment service

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

File: src/client/datascience/dataScienceSurveyBanner.ts [hunk 1]
```diff
@@ -115,20 +113,17 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
     public isEnabled(type: BannerType): boolean {
         switch (type) {
             case BannerType.InsidersNotebookSurvey:
-                if (this.useVSCodeNotebookEditorApi && this.applicationEnvironment.channel === 'insiders') {
+                if (this.applicationEnvironment.channel === 'insiders' && this.useVSCodeNotebookEditorApi) {
                     return this.isEnabledInternal(type);
```
Comment (greazer) [inline src/client/datascience/dataScienceSurveyBanner.ts:117]: Identical, just swapped right?

**Review by greazer — APPROVED** [review]
Approve, but I'd like at least one other real engineer to review. :)

File: src/client/datascience/dataScienceSurveyBanner.ts (hunk 1 above)
Comment (DavidKutu) [reply src/client/datascience/dataScienceSurveyBanner.ts:117]: yeah, I swapped it just so it looks the same as the other two

File: src/client/datascience/dataScienceSurveyBanner.ts [hunk 2]
```diff
@@ -115,20 +113,17 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
...
                 }
                 break;
             case BannerType.ExperimentNotebookSurvey:
-                if (
-                    this.applicationEnvironment.channel === 'stable' &&
-                    this.experimentService.inExperiment(Experiments.NativeNotebook)
```
Comment (DonJayamanne) [inline src/client/datascience/dataScienceSurveyBanner.ts:125]: Aah so the bug was we were not `awaiting` on the promise.

**Review by DonJayamanne — APPROVED** [review]

File: src/client/datascience/dataScienceSurveyBanner.ts (hunk 2 above)
Comment (DavidKutu) [reply src/client/datascience/dataScienceSurveyBanner.ts:125]: yeah, I didn't notice it was a promise, anyway, its cleaner to just use `useVSCodeNotebookEditorApi`

**Pull request merged**